### PR TITLE
[#1922] Remove roadmap from ProjectDetails

### LINKED ISSUE #1922: Remove roadmap in project details
Screen Shot 2021-07-04 at 4 44 22 PM

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #1922

### CODE REVIEW

File: src/containers/Projects/ProjectDetails/constants.ts [hunk 1]
```diff
@@ -33,12 +33,6 @@ export const projectDetailsTopic: ProjectTopicMap = {
     position: 1,
```
Comment (thesanjeevsharma) [inline src/containers/Projects/ProjectDetails/constants.ts:33]: Hey, `position` is not being used now I think. Can we also get rid of these in this PR?

Comment (jamessspanggg) [reply src/containers/Projects/ProjectDetails/constants.ts:33]: It is being used by the `ProjectDetailsSideMenu` component which uses the ordered version of the project topics.

Comment (thesanjeevsharma) [reply src/containers/Projects/ProjectDetails/constants.ts:33]: 🤔 that could have been achieved w/o position too, if we disable eslint rule on that file?

**Comment by thesanjeevsharma** [comment]
@jamessspanggg there's an icon for the roadmap. Should we remove that too?

File: src/containers/Projects/ProjectDetails/constants.ts (hunk 1 above)
Comment (angle943) [reply src/containers/Projects/ProjectDetails/constants.ts:33]: @thesanjeevsharma technically its unreliable to sort objects, although in practice it'll always be ordered the way we want it (after we disable es-lint). But just out of principle, i never like depending on the order of an object alone to dictate it's order

**Review by angle943 — APPROVED** [review]

**Pull request merged**